web/src/pages/user.jsx

```diff
@@ -6,7 +6,7 @@ import {useEffect, useState} from "react";
 import SongTile from "@/components/SongTile";
 import Fuse from "fuse.js";
 
-let songs = [];
+let songsCache = [];
```

Comment:
Is anything actually getting cached?

Reply:
yes, initial request to spotify api for web is being cached here instead of making the request over and over again every time the page loads, changed to songsCache to make variable name clear